test/monitor.go

```diff
@@ -0,0 +1,71 @@
+package test
```

Comment:
I would suggest that you put this in to separate package, so we can reuse it on other projects.